src/core/operations/Image.js

```diff
@@ -0,0 +1,36 @@
+import * as ExifParser from "exif-parser";
+import Utils from "../Utils.js";
+
+
+/**
+ * Image operations.
+ *
+ * @author tlwr [toby@toby.codes]
+ * @copyright Crown Copyright 2017
+ * @license Apache-2.0
+ *
+ * @namespace
+ */
+const Image = {
+    runEXIF(input, args) {
```

Comment:
It actually [already is an eslint rule](https://github.com/gchq/CyberChef/blob/master/.eslintrc.json#L68), however it's not very good at recognising functions declared as namespace properties. This is a [known bug](https://github.com/eslint/eslint/issues/5867) in eslint.